src/app/GitUI/CommandsDialogs/FormStash.cs

```diff
@@ -291,7 +291,7 @@ private void LoadGitItemStatuses(IReadOnlyList<GitItemStatus> gitItemStatuses)
         private void ResizeStashesWidth()
         {
             const int spacingBetweenLabelAndComboBox = 5;
-            Stashes.Width = toolStrip1.Width - spacingBetweenLabelAndComboBox - showToolStripLabel.Width;
+            Stashes.Size = new Size(toolStrip1.Width - DpiUtil.Scale(spacingBetweenLabelAndComboBox) - showToolStripLabel.Width, Stashes.Size.Height);
```

Comment:
Any reason why Height need to be set?

Reply:
Thank you for testing and fixing! 200% scaling is hardly testable on a Full HD display.

> In debugger I've noticed the value of Width was simply not updating

This seems to be an effect just in the debugger because `WndProc` is halted, too.

> I haven't dug into the specifics but it was like that before it stopped working.

The key is the missing `DpiUtil.Scale`. The following works and should be used for clarity:
```suggestion
            Stashes.Width = toolStrip1.Width - DpiUtil.Scale(spacingBetweenLabelAndComboBox) - showToolStripLabel.Width;
```

(I wonder why it worked before #12149 - most likely because of additional space.)
![image](https://github.com/user-attachments/assets/edfd99ab-675f-4750-9bda-56845b299014)

Reply:
Thanks for clarification reg. values in debugger. Changed to your proposed variant and tested it at 100% 150% and 200% and.... THIS... to see what's the experience to test scaling with 1080p screen... looked "gorgeous" on my 4K monitor :D

![image](https://github.com/user-attachments/assets/622d536b-265b-44eb-853d-ec6e204de2c6)